src/main/java/Avocado.java

```diff
@@ -0,0 +1,67 @@
+import java.util.Scanner;
+public class Avocado {
+    public static void main(String[] args) {
+        System.out.println("Hello from avocado");
+        System.out.println("How can I help you ?");
+        System.out.println("See you!");
+        Task[] list = new Task[100];
+        int i = 0;
+        int n = 0;
+        while (true) {
+            String line;
+            Scanner in = new Scanner(System.in);
+
+            line = in.nextLine();
+            if (line.equals("bye")) {
+                return;
+            }
+            String[] array;
+            Task t = new Task(line);
+            array = line.split(" ");
+            if (array[0].startsWith("mark")) {
+                list[Integer.parseInt(array[1]) - 1].setDone();
+                System.out.print(list[Integer.parseInt(array[1]) -1].taskDescription());
+            }
+            else if (array[0].startsWith("unmark")) {
+                list[Integer.parseInt(array[1]) - 1].setNotDone();
+                System.out.print( list[Integer.parseInt(array[1]) -1 ].taskDescription());
+            }
+            else if (array[0].startsWith("todo")) {
+                ToDo newtodo = new ToDo(line.substring(5));
+                list[i] = newtodo;
+                n++;
+                System.out.println(" [T]"  + list[i].taskDescription());
+                i++;
+
+
+            }
```

Comment:
For your conditionals, you could consider switching to using Egyptian style brackets to adhere to this [java coding standards](https://se-education.org/guides/conventions/java/basic.html#layout)